daemon/mgr/container.go

```diff
@@ -1696,6 +1686,15 @@ func (mgr *ContainerManager) execExitedAndRelease(id string, m *ctrd.Message) er
 	execConfig.Running = false
 	execConfig.Error = m.RawError()
 
+	if io := mgr.IOs.Get(id); io != nil {
+		if err := m.RawError(); err != nil {
+			fmt.Fprintf(io.Stdout, "%v\n", err)
+		}
+
+		// close io
+		io.Close()
+		mgr.IOs.Remove(id)
+	}
 	// TODO: GC invalid mgr.ExecProcess.
```

Comment:
Please rm this comment, we already have GC.

Reply:
Why comment the same thing twice? LOL :)

Reply:
Actually I did it only once 😢